Formula/freeswitch.rb

```diff
@@ -73,10 +74,6 @@ class Freeswitch < Formula
   uses_from_macos "libedit"
   uses_from_macos "zlib"
 
-  on_linux do
-    depends_on "util-linux"
-  end
-
```

Comment:
Is this only for sofia-sip?

Reply:
It's duplicated dependency. Have missed it when added `depends_on "util-linux"` for macOS.